### PR TITLE
RavenDB-6649

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: test/Tests.Infrastructure/RavenTestBase.cs [hunk 1]
```diff
@@ -276,6 +274,38 @@ protected void PutCommand(IDocumentSession session, object entity, string id)
             session.Advanced.RequestExecutor.Execute(putCommand, session.Advanced.Context);
         }
 
+        protected Task PutCommandAsync(IAsyncDocumentSession session, object entity, string id,
```
Comment (ppekrol) [inline test/Tests.Infrastructure/RavenTestBase.cs:277]: use DocumentStoreExtensions. We have Commands there

File: test/Tests.Infrastructure/RavenTestBase.cs [hunk 2]
```diff
@@ -243,26 +243,24 @@ public static void WaitForUserToContinueTheTest(DocumentStore documentStore, boo
...
         }
 
-        /// <summary>
-        /// Put command for new client tests
-        /// </summary>
-        /// <param name="session"></param>
-        /// <param name="entity"></param>
-        /// <param name="id"></param>
-        protected void PutCommand(IDocumentSession session, object entity, string id)
+        protected void PutCommand(IDocumentSession session, object entity, string id, bool skipMetadata = false)
```
Comment (ppekrol) [inline test/Tests.Infrastructure/RavenTestBase.cs:246]: use DocumentStoreExtensions. We have Commands there

Comment (fitzchak) [reply test/Tests.Infrastructure/RavenTestBase.cs:246]: Ok, I need to check that it can send a document without a metadata.

Comment (fitzchak) [reply test/Tests.Infrastructure/RavenTestBase.cs:246]: I see that you done this. Thanks.

File: test/Tests.Infrastructure/RavenTestBase.cs (hunk 1 above)
Comment (fitzchak) [reply test/Tests.Infrastructure/RavenTestBase.cs:277]: I see that you done this. Thanks.

**Pull request merged**